deploy.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 echo checking secure access
-openssl aes-256-cbc -K $encrypted_4d56ad91a2b0_key -iv $encrypted_4d56ad91a2b0_iv -in id_rsa.enc -out id_rsa -d
+- openssl aes-256-cbc -K $encrypted_4d56ad91a2b0_key -iv $encrypted_4d56ad91a2b0_iv -in id_rsa.enc -out id_rsa -d
```

Comment:
Is the `-` here a typo?

Reply:
Yeah it was my error from copying the line out of the travis file. I updated in the latest commit directly to master. Travis is publishing everything again, so all is good I think. Made the transition to travis.com yesterday so I think the auto publication of files upon merge should now be stable.